formats/service_signin.jsonnet

```diff
@@ -0,0 +1,86 @@
+(import "shared/default_format.jsonnet") + {
+  document_type: "service_signin",
+  definitions: {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      required: [
+        "choose_sign_in",
+        "create_new_account",
+      ],
+      properties: {
+        choose_sign_in: {
+          "$ref": "#/definitions/choose_sign_in",
+        },
+        create_new_account: {
+          "$ref": "#/definitions/create_new_account"
+        }
+      }
+    },
+    choose_sign_in: {
+      type: "object",
+      additionalProperties: false,
+      required: [
+        "title",
+        "slug",
+        "options",
+      ],
+      properties: {
+        title: {
+          type: "string",
+        },
+        slug: {
+          type: "string",
+          format: "uri",
+        },
+        description: {
+          "$ref": "#/definitions/body_html_and_govspeak",
+        },
+        options: {
+          type: "array",
+          additionalProperties: false,
+          required: [
+            "text",
+            "url",
+          ],
+          items: {
+            type: "object",
+            properties: {
+              text: {
+                type: "string",
+              },
+              url: {
```

Comment:
Are we expected this to be a full url or path?

Reply:
For external links we would expect a full url, and for internal links we'd expect base_path + slug.  I'll update the example to reflect the latter 👍